src/5e-SRD-Rules-Sections.json

```diff
@@ -0,0 +1,179 @@
--|----------|--------------------------------------------------|\n| Fast   | 400 feet             | 4 miles | 30 miles | −5 penalty to passive Wisdom (Perception) scores |\n| Normal | 300 feet             | 3 miles | 18 miles | —                                                |\n| Slow   | 200 feet             | 2 miles | 24 miles | Able to use stealth                              |\n\n#### Difficult Terrain\n\nThe travel speeds given in the Travel Pace table assume relatively simple terrain: roads, open plains, or clear dungeon corridors. But adventurers often face dense forests, deep swamps, rubble-filled ruins, steep mountains, and ice-covered ground—all considered difficult terrain.\n\nYou move at half speed in difficult terrain— moving 1 foot in difficult terrain costs 2 feet of speed—so you can cover only half the normal distance in a minute, an hour, or a day.\n\n### Special Types of Movement\n\nMovement through dangerous dungeons or wilderness areas often involves more than simply walking. Adventurers might have to climb, crawl, swim, or jump to get where they need to go.\n\n#### Climbing, Swimming, and Crawling\n\nWhile climbing or swimming, each foot of movement costs 1 extra foot (2 extra feet in difficult terrain), unless a creature has a climbing or swimming speed. At the GM’s option, climbing a slippery vertical surface or one with few handholds requires a successful Strength (Athletics) check. Similarly, gaining any distance in rough water might require a successful Strength (Athletics) check.\n\n#### Jumping\n\nYour Strength determines how far you can jump.\n\n***Long Jump.*** When you make a long jump, you cover a number of feet up to your Strength score if you move at least 10 feet on foot immediately before the jump. When you make a standing long jump, you can leap only half that distance. Either way, each foot you clear on the jump costs a foot of movement.\n\nThis rule assumes that the height of your jump doesn’t matter, such as a jump across a stream or chasm. At your GM’s option, you must succeed on a DC 10 Strength (Athletics) check to clear a low obstacle (no taller than a quarter of the jump’s distance), such as a hedge or low wall. Otherwise, you hit it.\n\nWhen you land in difficult terrain, you must succeed on a DC 10 Dexterity (Acrobatics) check to land on your feet. Otherwise, you land prone.\n\n***High Jump.*** When you make a high jump, you leap into the air a number of feet equal to 3 + your Strength modifier if you move at least 10 feet on foot immediately before the jump. When you make a standing high jump, you can jump only half that distance. Either way, each foot you clear on the jump costs a foot of movement. In some circumstances, your GM might allow you to make a Strength (Athletics) check to jump higher than you normally can.\n\nYou can extend your arms half your height above yourself during the jump. Thus, you can reach above you a distance equal to the height of the jump plus 1½ times your height.\n",
```

Comment:
Should the double-quotes be curved, or straight (and escaped)? Some renderers couldn't probably render this without issues. Are we providing the content in a Unicode character set, or in ASCII, if it's ASCII, it should definitely be switched to UTF-8 (or 16 or 32, but UTF-8 would provide the best bytes per character ration in this entire file).

Reply:
Straight and escaped, please.